portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase

```diff
@@ -76,13 +76,46 @@ definition {
 	}
 
 	@description = "Verify that a Numeric Field can be required"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeRequired {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132832 CanBeRequired pending implementation
+		
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Numeric",
+			formPageNumber = "0");
+
+		FormFields.enableSwitcher(fieldName = "Required Field");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormViewBuilder.validateFieldLabelRequired(fieldLabel = "Numeric");
+
+		FormPortlet.waitFormRenderer(text = "Submit");
+
+		FormPortlet.submitWithValidationError(validationErrorMessage = "This field is required.");
+
+		FormFields.editText(
+			fieldName = "Numeric",
+			fieldValue = "123");
+
+		FormPortlet.submitSuccessfully();
+
+
```

Comment:
Great job @leticiachiu!

The only thing missing was the `Assert the Success Page' step.

```
FormPortlet.viewSuccessPage(
	successPageDescription = "Your information was successfully received. Thank you for filling out the form.",
	successPageTitle = "Thank you.");
```

Can you please add this and resolve the conflicts?

Thanks!

Reply:
hello, carol! this macro actually does the assert of success too:
  `macro submitSuccessfully {
		FormPortlet.submit();

		WaitForElementPresent(locator1 = "Message#SUCCESS_DISMISSIBLE");

		Alert.viewSuccessMessage();
	}`

Reply:
Thanks for the explanation, @leticiachiu ! Can you resolve the conflicts so I can forward the PR? Thanks!